Enterprise-CMCS/macpro-mdct-mcr
# Add new get started content to MCPAR initial page

## Summary

Add some new content to the MCPAR Get Started page. Adds some new sections to the MCPAR verbiage that will only render if they exist in the data.

At some point we should add types to the verbiage—I might do this today or tomorrow, but in a separate PR.

### Related ticket
[2106](https://qmacbis.atlassian.net/browse/MDCT-2106)

### How to test
1. Log in as a state user
2. Select MCPAR from the home dashboard
3. Verify new content matching ticket information is displayed.

### Important updates
N/A

### Author checklist
